src/components/atoms/bar.spec.ts

```diff
@@ -0,0 +1,25 @@
+
+
+describe("my test case", () => {
+    it('Should work', () => {
+        const myValueF = (a, b) => {
+            return a + b;
+        }
+
```

Comment:
Testing a function created inside the test is pointless. Also name it better.

Try using only one assertion per test case